pkg/sdk/warehouses.go

```diff
@@ -52,6 +52,52 @@ var (
 	WarehouseSizeX6Large  WarehouseSize = "X6LARGE"
 )
 
+func ToWarehouseSize(s string) (WarehouseSize, error) {
+	s = strings.ToUpper(s)
+	switch s {
+	case "XSMALL":
+		return WarehouseSizeXSmall, nil
+	case "X-SMALL":
+		return WarehouseSizeXSmall, nil
```

Comment:
```suggestion
	case "XSMALL", "X-SMALL":
		return WarehouseSizeXSmall, nil
```
you can refactor to have one entry per size.